561.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var arrayPairSum = function (nums) {
+  const sorted = [...nums].sort((a, b) => a - b);
+  let maxNum = 0;
```

Comment:
I think `maxSum` sounds better
```suggestion
  let maxSum = 0;
```